server/src/email-password/authenticate.ts

```diff
@@ -1,66 +1,72 @@
-import { fromEvent, FunctionEvent } from 'graphcool-lib'
-import { GraphQLClient } from 'graphql-request'
-import * as bcrypt from 'bcryptjs'
+import { fromEvent, FunctionEvent } from 'graphcool-lib';
+import { GraphQLClient } from 'graphql-request';
+import * as bcrypt from 'bcryptjs';
 
 interface User {
-  id: string
-  password: string
+  id: string;
+  password: string;
+  showOnboarding: boolean;
 }
 
 interface EventData {
-  email: string
-  password: string
+  email: string;
+  password: string;
 }
 
-const SALT_ROUNDS = 10
+const SALT_ROUNDS = 10;
 
 export default async (event: FunctionEvent<EventData>) => {
-  console.log(event)
+  console.log(event);
 
   try {
-    const graphcool = fromEvent(event)
-    const api = graphcool.api('simple/v1')
+    const graphcool = fromEvent(event);
+    const api = graphcool.api('simple/v1');
 
-    const { email, password } = event.data
+    const { email, password } = event.data;
 
     // get user by email
-    const user: User = await getUserByEmail(api, email)
-      .then(r => r.User)
+    const user: User = await getUserByEmail(api, email).then(r => r.User);
 
     // no user with this email
     if (!user) {
-      return { error: 'Invalid credentials!' }
+      return { error: 'Invalid credentials!' };
     }
 
     // check password
-    const passwordIsCorrect = await bcrypt.compare(password, user.password)
+    const passwordIsCorrect = await bcrypt.compare(password, user.password);
     if (!passwordIsCorrect) {
-      return { error: 'Invalid credentials!' }
+      return { error: 'Invalid credentials!' };
     }
 
     // generate node token for existing User node
-    const token = await graphcool.generateNodeToken(user.id, 'User')
+    const token = await graphcool.generateNodeToken(user.id, 'User');
 
-    return { data: { id: user.id, token} }
+    return {
+      data: { id: user.id, token, showOnboarding: user.showOnboarding },
+    };
   } catch (e) {
-    console.log(e)
-    return { error: 'An unexpected error occured during authentication.' }
+    console.log(e);
+    return { error: 'An unexpected error occured during authentication.' };
```

Comment:
looks like your prettier settings are different than your teammates. Or maybe their settings are different. Please make sure that all of you have the same prettier settings.

Reply:
Prettier settings are consistent amongst the group.  This page got restyled because its a template file we downloaded.  I had to make changes in it for the first time yesterday and that resulted in Prettier being applied to it.

Reply:
good to know